.github/workflows/run-qit.yml

```diff
@@ -0,0 +1,85 @@
+name: Run QIT
+
+# **What it does**: Runs a suite of QIT tests for the extension.
+# **Why we have it**: To be able to check QIT compatibility at once. For example, to test a specific branch, or upcoming release.
+
+on:
+  workflow_dispatch:
+    inputs:
+      wait:
+        description: 'Should wait for results'
+        default: false
+        type: boolean
+      # Configure which tests to run.
+      test-activation:
+        description: 'Should activation be tested?'
+        required: true
+        default: true
+        type: boolean
+      test-security:
+        description: 'Should security be tested?'
+        required: true
+        default: true
+        type: boolean
+      test-phpstan:
+        description: 'Should phpstan be tested?'
+        required: true
+        default: true
+        type: boolean
+      test-api:
+        description: 'Should API be tested?'
+        required: true
+        default: true
+        type: boolean
+      test-e2e:
+        description: 'Should E2E be tested? (takes a lot of time)'
+        required: true
+        default: false
+        type: boolean
+
+      # Advanced customization.
+      ignore-fail:
+        description: Should pass even if any awaited test fails.
+        required: false
+        default: false
+        type: boolean
+      options:
+        description: 'Additional options for `qit` command, like `--optional_features=hpos`.'
+        required: false
+
+  # Used to test this PR, to be removed after before merging.
+  push:
+    branches:
+      - dev/qit-workflow
+
+jobs:
+  build:
+    name: Build extension
+    uses: ./.github/workflows/build.yml
+    secrets: inherit
+  qit-tests:
+    name: Run QIT Tests
+    runs-on: ubuntu-20.04
+    needs: build
+    steps:
+      - name: Download artifact
+        uses: actions/download-artifact@v3
+        with:
+          name: google-listings-and-ads.zip
+      - name: Delegate QIT Tests
```

Comment:
💅  Even if it's true that we delegate the QIT Test to the Grow Repo... I would just call this `Run QIT Tests` as it's the final goal of what we are doing. The same way we do for other workflows. We don't call them Delegate Get Woocommerce Matrix or Delegate Run PHP tests.